src/setupTests.ts

```diff
@@ -0,0 +1,8 @@
+// react-testing-library renders your components to document.body,
```

Comment:
This file used to be in `./internals`. With CRA, how does jest know to include this? Is it bundled with webpack for production (affecting bundle size by including test dependencies)?